src/operator/nn/mkldnn/mkldnn_base.cc

```diff
@@ -151,9 +151,12 @@ const mkldnn::memory *GetWeights(const NDArray &arr,
     LOG(FATAL) << "The weight array has an unsupported number of dimensions";
     return nullptr;
   }
-  if (mem == nullptr)
+  if (mem == nullptr) {
     mem = arr.GetMKLDNNDataReorder(target_pd);
-  if (mem->get_primitive_desc() == target_pd) return mem;
+  }
+  if (mem->get_primitive_desc() == target_pd) {
+    return mem;
+  }
```

Comment:
please avoid changing the way the original code was written if you are not modifying it.